src/main/webapp/resources/js/pages/projects/processing/ProcessingCoverage.jsx

```diff
@@ -0,0 +1,161 @@
+import React from "react";
+import {
+  Button,
+  Card,
+  Col,
+  Form,
+  InputNumber,
+  Modal,
+  notification,
+  Row,
+  Statistic,
+  Typography,
+} from "antd";
+import isNumeric from "antd/es/_util/isNumeric";
+import {
+  fetchProcessingCoverage,
+  updateProcessingCoverage,
+} from "../../../apis/projects/settings";
+
+export function ProcessingCoverage({ projectId, canManage }) {
+  const [visible, setVisible] = React.useState(false);
+  const [coverage, setCoverage] = React.useState({});
+  const [form] = Form.useForm();
+
+  const getCoverage = React.useCallback(async () => {
+    const { minimum, maximum, genomeSize } = await fetchProcessingCoverage(
+      projectId
+    );
+    setCoverage({
+      minimum: minimum > -1 ? minimum : "No set",
```

Comment:
Internationlize the "Not Set" here and below

Reply:
Updated in 9e7b2d7